core/src/main/java/org/keycloak/representations/UserInfo.java

```diff
@@ -277,11 +277,11 @@ public void setPhoneNumberVerified(Boolean phoneNumberVerified) {
         this.phoneNumberVerified = phoneNumberVerified;
     }
 
-    public AddressClaimSet getAddress() {
+    public Map<String, Object> getAddress() {
```

Comment:
Is it possible to make sure that getter/setter are backwards compatible? Something similar like you did for `IDToken` perhaps might work here as well?